nifi-nar-bundles/nifi-hbase-bundle/nifi-hbase-processors/src/main/java/org/apache/nifi/hbase/ScanHBase.java

```diff
@@ -209,6 +209,17 @@
             .addValidator(StandardValidators.CHARACTER_SET_VALIDATOR)
             .build();
 
+    static final PropertyDescriptor BLOCK_CACHE = new PropertyDescriptor.Builder()
+            .displayName("Block Cache Set")
```

Comment:
I think it would be better to just name it "Block Cache"

Reply:
made change.